test/__mocks__/audio-effects.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env jest */
+export default class MockAudioEffects {
+    static get effectTypes () { // @todo can this be imported from the real file?
```

Comment:
Probably? Importing and then returning `AudioEffects.effectTypes` seems like it should work?